LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Chiibeme
```

Comment:
If you're going to add a license it should be copyright this year and Tapendium :)

Reply:
Yup! 😅